cfn-templates/cid-cfn.yml

```diff
@@ -108,26 +108,26 @@ Parameters:
   DeployCUDOSDashboard:
     Type: String
     Description: Deploy CUDOS Dashboard
-    Default: "no"
+    Default: "yes"
     AllowedValues: ["yes", "no"]
   DeployCostIntelligenceDashboard:
     Type: String
     Description: Deploy Cost Intelligence Dashboard
-    Default: "no"
+    Default: "yes"
     AllowedValues: ["yes", "no"]
   DeployKPIDashboard:
     Type: String
     Description: Deploy KPI Dashboard
-    Default: "no"
+    Default: "yes"
     AllowedValues: ["yes", "no"]
   DeployTAODashboard:
     Type: String
-    Description: Deploy Trusted Advisor Organisational Dashboard (TAO) - You need Optimisation Data Collection Lab to be installed
+    Description: ADDITIONAL - Deploy Trusted Advisor Organisational Dashboard (TAO) - WARNING! You need Optimisation Data Collection Lab to be installed https://wellarchitectedlabs.com/cost/300_labs/300_optimization_data_collection/
```

Comment:
Before deploying this dashboard, you need Optimisation Data Collection Lab to be installed first https://wellarchitectedlabs.com/cost/300_labs/300_optimization_data_collection/